.github/workflows/snap-creation.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         sudo apt install --yes git
         sudo snap install snapcraft --classic
         export ROS_DISTRO=melodic
-        git clone --recurse-submodules --depth 3 --single-branch --branch ${{ github.ref }} https://github.com/cyberbotics/webots.git
+        git clone --recurse-submodules --depth 3 --single-branch --branch ${{ github.event.release.tag_name }} https://github.com/cyberbotics/webots.git
```

Comment:
Hi, note that 'release' is not an event that is triggering this workflow:
```
on:
  pull_request:
    types: [opened, synchronize, reopened, labeled, unlabeled]
  schedule:
  - cron: "0 23 * * *"
  push:
    tags:
      - '*'
````

Reply:
Yes, thank you. I already fixed the problem by using:
```
git clone --recurse-submodules --depth 3 --single-branch --branch ${GITHUB_REF#refs/tags/} https://github.com/cyberbotics/webots.git
```
instead.